content/documentation/getting-started/index.adoc

```diff
@@ -319,11 +319,35 @@ auth:
     ldap_role_filter: .*xyz.*
 ----
 
-The mandatory  configurations are:
+_ldap_host_: This parameter specifies the host IP of the LDAP server.
+_ldap_port_: This parameter specifies the host port of the LDAP server.
+_ldap_use_ssl_: is to enable the usage of SSL certificate in the LDAP request
+_ldap_insecure_skip_verify_: This parameter is to skip verification of the CA Directory.
+_ldap_role_filter_: This configuration is used to filter the user roles based on the regular expression provided.
+_ldap_user_filter_: This is used to search for the given user name. If the user name is part of Common Name(CN). The filter will be like `(cn=%s)`
```

Comment:
`If the user name is part of Common Name(CN).` is an incomplete sentence. Just replace the period (`.`) with a comma (`,`) and use lowercase "The" and the sentence will be correct:

```
If the user name is part of Common Name(CN), the filter will be like `(cn=%s)`
```